lib/maintenance/MaintenanceManager.js

```diff
@@ -64,7 +64,13 @@ class MaintenanceManager {
       state: CONST.OPERATION.IN_PROGRESS
     };
     return Repository
-      .findOne(CONST.DB_MODEL.MAINTENANCE_DETAIL, criteria);
+      .findOne(CONST.DB_MODEL.MAINTENANCE_DETAIL, criteria)
+      .then(info => info === null ? null : _.set(_.clone(info), 'broker_update_initiated', this.isBrokerUpdateInitiated(info)));
```

Comment:
can we have this as a constant `broker_update_initiated` instead of duplicating it ?

Reply:
This attrib removed completely with newer impl.